client/src/components/Tickets/TicketCreateForm.js

```diff
@@ -222,21 +226,40 @@ function TicketCreateForm({
                     Ticket creating error: {creatingErrorMessage}
                   </Alert>
                 )}
-                <Button
-                  type="submit"
-                  color="primary"
-                  disabled={!isValid || isSubmitting || creating}
-                >
-                  {creating && (
-                    <Spinner
-                      size="sm"
-                      color="white"
-                      data-testid="creating-spinner"
-                      className="ticket-create-spinner"
-                    />
-                  )}
-                  Add Ticket
-                </Button>
+                {isAuthenticated && (
```

Comment:
rather than have an `isAuthenticated` wrapper, can we apply that bool to determine the value/result of the ReCaptcha state? As in, if `! isAuthenticated`, then the value of the recaptcha response will apply towards `disabled` state of the button

Reply:
So  I think you're talking about line 250 and Recaptcha response will apply as  `setDisableSubmit(false);` depending on Recaptcha response. (The reason I have both `isAuthenticated` and` !isAuthenticated` is by default unauthenticated page need to be disabled and by default authenticated page button need to submittable regardless recaptcha)

Reply:
Are we able to combine the recaptcha value with the formik validation schema?

Reply:
@rstijerina @xandroc Currently I tested with the backend side and frontend it should work as expected. if you delete the Recaptcha key it should not show Recaptcha but the button should be clickable and it has a Recaptcha key then it will show Recaptcha and unclickable until Recaptcha is validated. I add to Recaptcha value on yup() and defaultValue, assign to name but still, it will be a better approach to implement as Recaptcha `.required` and replace the current boolean logic `disableSubmit`.